pkg/plugins/resources/yaml/condition_test.go

```diff
@@ -53,27 +59,30 @@ github:
 `,
 			wantResult: true,
 			wantMockState: text.MockTextRetriever{
-				Location: "test.yaml",
-			},
-		},
-		{
-			name: "Failing case with keyonly and input source",
-			spec: Spec{
-				File:    "test.yaml",
-				Key:     "github.country",
-				KeyOnly: true,
-			},
-			inputSourceValue: "",
-			mockReturnedContent: `---
+				Contents: map[string]string{
+					"test.yaml": `---
 github:
   owner: olblak
   repository: charts
 `,
-			wantResult: false,
-			wantMockState: text.MockTextRetriever{
-				Location: "test.yaml",
+				},
 			},
 		},
+		// 		{
+		// 			name: "Failing case with keyonly and input source",
```

Comment:
What is the reason to comment this test?

Reply:
Because this one and the following were resisting me ^^